documentation/asciidoc/topics/proc_restoring_cluster.adoc

```diff
@@ -0,0 +1,51 @@
+[id='restoring_clusters-{context}']
+= Restoring {brandname} Clusters
+Restore {brandname} cluster state from backup archives.
+When you restore cluster state, {ispn_operator} applies all caches and data from the backup archive on to the target cluster.
+
+.Prerequisites
+
+* Create a {brandname} cluster of {datagridservice} nodes where you want to restore state.
++
+[IMPORTANT]
+====
+The operation to restore {brandname} overwrites state on the target cluster.
+To prevent data loss, you should ensure there are no active client connections to the cluster where you want to restore state.
```

Comment:
Similar to the comments about data loss for backups, however it's probably more important in the Restore case as data restored from an archive could overwrite a client modification `k=2` that is received during the restore, but before `k=1` is processed during the restore.